script/index.coffee

```diff
@@ -516,13 +513,14 @@ api.wrap = (user, main=true) ->
         flags.dropsEnabled = false
         flags.classSelected = false
         flags.levelDrops = {}
-        # Award an achievement if this is their first Rebirth, or if they made it further than last time
+        # Award an achievement if this is their first Rebirth, or if they made it further than last time, or if they hit over 100
         if not (user.achievements.rebirths)
           user.achievements.rebirths = 1
           user.achievements.rebirthLevel = lvl
-        else if (lvl > user.achievements.rebirthLevel or lvl is 100)
-          user.achievements.rebirths++
+        else if (lvl > user.achievements.rebirthLevel)
           user.achievements.rebirthLevel = lvl
+        if (lvl >= 100)
```

Comment:
looks like there's actually another problem with it, too...

rebirths under 100 aren't going to stack.